build/olm_builder.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
You have changed the path of this Dockerfile build/docker-image/Dockerfile, change the Makefile target for that

Reply:
the makefile target that triggers this script correctly points to it:
`build-and-release-olm-operator:
	./build/olm_builder.sh
`

also, the make targets for building images are also pointing to right Dockerfiles.